packages/pn-personagiuridica-webapp/src/components/IntegrazioneApi/VirtualKeyContextMenu.tsx

```diff
@@ -132,8 +109,57 @@ const VirtualKeyContextMenu: React.FC<Props> = ({ data, keys, handleModalClick }
             <Delete sx={{ mr: 1 }} />
             {t('button.elimina', { ns: 'common' })}
           </MenuItem>
-        )}
+        ));
+      }
+      return {
```

Comment:
You don't need to return anything, menuElements array has already been populated ad this point and you can verify if it's empty using the .length property.

Reply:
done